plugins/CoreAdminHome/templates/home.twig

```diff
@@ -21,9 +21,11 @@
             <div class="col s12 {% if isFeedbackEnabled %}m4{% else %}m6{% endif %}">
                 <div piwik-widget-loader='{"module":"CoreHome","action":"getSystemSummary"}'></div>
             </div>
-            <div class="col s12 {% if isFeedbackEnabled %}m4{% else %}m6{% endif %}">
-                <div piwik-widget-loader='{"module":"Installation","action":"getSystemCheck"}'></div>
-            </div>
+            {% if hasDiagnostics %}
```

Comment:
It is ok to merge although ideally we would need to calculate the column width correctly so the widgets use full width. eg when feedback is disabled, and system check is disabled, system summary would have m12 or s12 class. When only system summary is disabled, both feedback and system summary would have m6 etc